### PR TITLE
Update documentation for colored coin API

### PR DESCRIPTION
**Pull request by Yamaguchi**
This PR updates documentation for RocksDB schema and  Esplora REST API to support colored coin.

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
+# Esplora HTTP API
+
+JSON over RESTful HTTP. Amounts are always represented in satoshis.
```
Comment (azuchi) [inline doc/API.md:3]:
```suggestion
JSON over RESTful HTTP. Amounts are always represented in tapyrus.
```

File: doc/API.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+The blockstream.info public APIs are available at:
+- Bitcoin: https://blockstream.info/api/
+- Bitcoin Testnet: https://blockstream.info/testnet/api/
+- Liquid: https://blockstream.info/liquid/api/
```
Comment (azuchi) [inline doc/API.md:8]: A list of these APIs is not necessary.

File: doc/API.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+
+> In this example, there are transactions weighting a total of 102,131 vbytes that are paying more than 53 sat/vB,
+110,990 vbytes of transactions paying between 38 and 53 sat/vB, 138,976 vbytes paying between 34 and 38, etc.
+
+
+### `GET /mempool/txids`
+
+Get the full list of txids in the mempool as an array.
+
+The order of the txids is arbitrary and does not match bitcoind's.
```
Comment (azuchi) [inline doc/API.md:218]:
```suggestion
The order of the txids is arbitrary and does not match tapyrusd's.
```

File: doc/API.md [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+
+### `GET /tx/:txid/hex`
+### `GET /tx/:txid/raw`
+
+Returns the raw transaction in hex or as binary data.
+
+### `GET /tx/:txid/merkleblock-proof`
+
+Returns a merkle inclusion proof for the transaction using
+[bitcoind's merkleblock](https://bitcoin.org/en/glossary/merkle-block) format.
```
Comment (azuchi) [inline doc/API.md:40]:
```suggestion
[tapyrusd's merkleblock](https://bitcoin.org/en/glossary/merkle-block) format.
```

File: doc/API.md [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+  - `confirmed` (boolean)
+  - `block_height` (available for confirmed transactions, `null` otherwise)
+  - `block_hash` (available for confirmed transactions, `null` otherwise)
+  - `block_time` (available for confirmed transactions, `null` otherwise)
+
+## Block format
+
+- `id`
+- `height`
+- `version`
```
Comment (azuchi) [inline doc/API.md:324]: In tapyrus, use `features` instead of `version`.

File: doc/API.md [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+- `timestamp`
+- `tx_count`
+- `size`
+- `weight`
+- `merkle_root`
+- `im_merkle_root`
+- `previousblockhash`
+- `mediantime` (median time-past)
```
Comment (azuchi) [inline doc/API.md:332]:
```suggestion
- `time`
```

File: doc/API.md [hunk 7]
```diff
@@ -0,0 +1,335 @@
...
+- `signature`
+
```
Comment (azuchi) [inline doc/API.md:334]: In addition to the above, there are `xfield type` and `xfield`.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: doc/API.md (hunk 6 above)
Comment (Yamaguchi) [reply doc/API.md:320]: Field `timestamp` should be renamed to `time`?

**Review by azuchi — APPROVED** [review]

**Pull request merged**